zerver/tests/test_auth_backends.py

```diff
@@ -1884,12 +1884,16 @@ class AppleAuthMixin:
     AUTH_FINISH_URL = "/complete/apple/"
     CONFIG_ERROR_URL = "/config-error/apple"
 
-    def generate_id_token(self, account_data_dict: Dict[str, str]) -> str:
+    def generate_id_token(self, account_data_dict: Dict[str, str], aud: Optional[str]=None) -> str:
         payload = account_data_dict
 
         # This setup is important because python-social-auth decodes `id_token`
         # with `SOCIAL_AUTH_APPLE_CLIENT` as the `audience`
         payload['aud'] = settings.SOCIAL_AUTH_APPLE_CLIENT
+
+        if aud is not None:
```

Comment:
What is `aud`?  We should expand it if it's an abbreviation.

Reply:
Oh, `audience`; yeah, let's make the parameter to the function `audience`, at least, even if we need to put it into a dict as `aud`.  More readable that way.